styles.css

```diff
@@ -322,7 +322,7 @@ li {
 }
 
 .project-1-para {
```

Comment:
- [ ] Great job implementing the about me section 👍. However, there are still some issues in your "recent works" section. As mentioned by the previous reviewer, please make sure your portfolio looks identical to the figma template. Your "My recent works" section is currently misaligned and the image seems to be too big for mobile. This also seems to be creating a horizontal scrollbar:

Your design: 
![multi](https://user-images.githubusercontent.com/111214189/229313901-1377cbac-2943-478c-b015-8eafd228a31f.png)

Figma template: 
![templateworks](https://user-images.githubusercontent.com/111214189/229313977-649cadd1-aaa5-4fcc-b2aa-f6b05036fe49.png)

Kindly make sure your section is properly aligned like in the figma template and your image fits the mobile version screen. This will ensure that your portfolio looks professional and that you have great attention to detail 👍 .
(Hint: you can take a look at your .experience-container class and try to use grid justify-content: center to align it, and 
 flexbox align-items: center in your .project-1 class. Make sure the container fits the screen and has the same margin as in the template without a horizontal scroll bar)

- [ ] As mentioned by the previous reviewer, kindly make sure your works section has the correct margins and padding, and is also correctly aligned:

Your design:
![missal](https://user-images.githubusercontent.com/111214189/229314418-e770f813-9126-49f1-9cec-14317a6badf8.png)


Figma template:
![figmacard](https://user-images.githubusercontent.com/111214189/229314292-e4114161-f68c-4b98-a56f-96865fc05052.png)

This is not only a requirement, but fixing this issue will ensure your project looks professional and that you know how to follow a figma template design 👍